elpy/blackutil.py

```diff
@@ -30,6 +30,12 @@ def parse_version(*arg, **kwargs):
         black = None
     else:
         import black
+        current_version = parse_version(black.__version__)
+        if current_version >= parse_version("21.5b1"):
+            from black.files import find_pyproject_toml
+        elif current_version >= parse_version("20.8b0"):
```

Comment:
Did `find_project_toml` not exist before this version?
If it didn't then we should have `find_project_toml = None` as an else branch so that we don't throw an error later down the road since `find_project_toml` won't be defined.

Reply:
Yes, it was introduced about two years ago. I'll set it to None in the else block.